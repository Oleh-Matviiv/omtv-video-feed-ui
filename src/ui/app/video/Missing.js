import React from 'react';
import styled from 'styled-components';

export const Div = styled.div.attrs({
  className: 'video-missing',
})`
  grid-area: play;
  background: #fafafa;
  display: flex;
  justify-content: center;
  padding: 7rem 0;
  color: #b4bbc1;
`;

const Missing = () => <Div>:/ video is missing</Div>;

export default Missing;
