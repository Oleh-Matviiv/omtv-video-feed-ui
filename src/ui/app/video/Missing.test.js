import React from 'react';
import { mount, shallow } from 'enzyme';
import 'jest-styled-components';

import Missing, { Div } from './Missing';

describe('snapshots', () => {
  it('Missing', () => {
    const wrapper = shallow(
      <Missing />
    );

    expect(wrapper).toMatchSnapshot();
  });
  it('Div', () => {
    const wrapper = shallow(
      <Div>children</Div>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
