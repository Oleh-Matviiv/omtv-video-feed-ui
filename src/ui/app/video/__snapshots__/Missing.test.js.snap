// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots Div 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [
        Object {
          "className": "video-missing",
        },
      ],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  grid-area: play;
  background: #fafafa;
  display: flex;
  justify-content: center;
  padding: 7rem 0;
  color: #b4bbc1;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  children
</StyledComponent>
`;

exports[`snapshots Missing 1`] = `
<styled.div>
  :/ video is missing
</styled.div>
`;
